articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,188 @@
+
+4. **Flutter DevTools**
+
+It is a tooling web suite for Flutter and Dart developers comprising a layout inspection tools, performance tools and memory tools. It contains all the debugging tools that you need to be an efficient and effective Flutter developer.
+
+Some of tools in the [Flutter DevTools web suite](https://flutter.dev/docs/development/tools/devtools/overview) include:
+
+1. Flutter Inspector - It is used to diagnose and debug layout issues.
+
+2. Timeline - It diagnoses performance problems which are the most common kinds during development. The Timeline shows the amount of time taken by each widget.
+
+
+5. **Design Tools**
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It ensures for a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- [Sketch.](https://www.sketch.com/) This is however only available for Mac.
+- [Adobe XD.](https://www.sketch.com/)
+- [Figma.](https://www.figma.com/) 
+- [Flutter Studio.](https://flutterstudio.app/)
+
+After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code. Supernova is compatible with the design tools mentioned above. 
+
+### Flutter Developer Skills
+
+1. **Dart Programming Language**
+
+`Dart` is a client-based programming language that is scalable and can be used to write simple scripts. It is an object-oriented and dynamic language. Google developed dart. It is the primary language used in Flutter development.
+
+2. **Knowledge on Agile Methodology and Waterfall for Software Development**
+
+A software development method is dividing the software development process into different phases. It enhances product design and resource utilization ***for example*** money and time. 
+
+The most commonly used methodologies are `Waterfall Methodology` and `Agile Methodology`. A Flutter developer needs to familiarize themselves with the software methodologies to work with ease in a software development team.
+
+`Waterfall Methodology` is a system development life cycle model used to create a system with a linear and sequential approach.
+`Agile Methodology` is also a system development life cycle model that implements the practice of iteration during testing and development throughout the cycle.
+
+3. **Prior knowledge on a Native Language**
+
+The native languages for mobile application development include:
+
+1. `Swift` for iOS applications.
+
+2. `Java` and `Kotlin` for Android applications.
+
+Understanding these languages will make it easier to become a flutter developer.
+
+### Mistakes to Avoid When Learning Flutter Development
```

Comment:
Could you adjust the sub sections to align with the section heading.
`### Mistakes to Avoid When Learning Flutter Development
- **Learn native languages, not just Flutter and Dart.**]
- **Learn native languages, not just Flutter and Dart.*`
It should be
`### Mistakes to Avoid When Learning Flutter Development
- **Ignoring native languages.**]